### PR TITLE
[Glimmer2] Port {{partial}} test to ember-glimmer.

### PR DESCRIPTION
**Pull request by jheth**
This converts the existing `{{partial}}` test to glimmer.

As noted, `partial` is not support in Glimmer 2 so these are tagged to `@htmlbars`

The one addition to the tests is calling `assertStableRerender`

Issue #13127

### CODE REVIEW

**Comment by jheth** [comment]
All test method descriptions remained the same.

File: packages/ember-htmlbars/tests/helpers/partial_test.js [hunk 1]
```diff
@@ -1,105 +0,0 @@
...
-  },
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-    Ember.lookup = originalLookup;
-    view = owner = null;
-  }
-});
-
-QUnit.test('should render other templates registered with the container', function() {
```
Comment (jheth) [inline packages/ember-htmlbars/tests/helpers/partial_test.js]: Replaced by Line [6](https://github.com/emberjs/ember.js/pull/13199/files#diff-6200fd37f2358aea2699a69836cc0f2bR6)

File: packages/ember-htmlbars/tests/helpers/partial_test.js [hunk 2]
```diff
@@ -1,105 +0,0 @@
...
-    [OWNER]: owner,
-    template: compile('This {{partial "subTemplateFromContainer"}} is pretty great.')
-  });
-
-  runAppend(view);
-
-  equal(trim(view.$().text()), 'This sub-template is pretty great.');
-});
-
-QUnit.test('should render other slash-separated templates registered with the container', function() {
```
Comment (jheth) [inline packages/ember-htmlbars/tests/helpers/partial_test.js]: Replaced by Line [16](https://github.com/emberjs/ember.js/pull/13199/files#diff-6200fd37f2358aea2699a69836cc0f2bR16)

File: packages/ember-htmlbars/tests/helpers/partial_test.js [hunk 3]
```diff
@@ -1,105 +0,0 @@
...
-    [OWNER]: owner,
-    template: compile('This {{partial "child/subTemplateFromContainer"}} is pretty great.')
-  });
-
-  runAppend(view);
-
-  equal(trim(view.$().text()), 'This sub-template is pretty great.');
-});
-
-QUnit.test('should use the current view\'s context', function() {
```
Comment (jheth) [inline packages/ember-htmlbars/tests/helpers/partial_test.js]: Replaced by Line [26](https://github.com/emberjs/ember.js/pull/13199/files#diff-6200fd37f2358aea2699a69836cc0f2bR26)

File: packages/ember-htmlbars/tests/helpers/partial_test.js [hunk 4]
```diff
@@ -1,105 +0,0 @@
...
-    firstName: 'Kris',
-    lastName: 'Selden'
-  }));
-
-  runAppend(view);
-
-  equal(trim(view.$().text()), 'Who is Kris Selden?');
-});
-
-QUnit.test('Quoteless parameters passed to {{template}} perform a bound property lookup of the partial name', function() {
```
Comment (jheth) [inline packages/ember-htmlbars/tests/helpers/partial_test.js]: Replaced by Line [36](https://github.com/emberjs/ember.js/pull/13199/files#diff-6200fd37f2358aea2699a69836cc0f2bR36)

**Comment by wycats** [comment]
@jheth Thanks so much for doing the initial port.

A couple things:

- Can you add the `U` and `R` steps? For example, you will want to test updating the partial name and elements of the context used in the partial.
- There are a couple of places where the old `{{template}}` name is used in the test name and should be changed to `{{partial}}`.

There are some similar problems in the dynamic component ("component helper") case. You can have a look at https://github.com/emberjs/ember.js/blob/master/packages/ember-glimmer/tests/integration/components/dynamic-components-test.js.

**Comment by chancancode** [comment]
superseded by #13199

**Pull request closed without merging**